customers-service/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM java:openjdk-8u111-alpine
-RUN apk --no-cache add curl
-CMD java ${JAVA_OPTS} -jar customers-service.jar
-HEALTHCHECK --start-period=30s --interval=5s CMD curl -f http://localhost:8080/health || exit 1
-COPY build/libs/customers-service.jar .
+ARG baseImageVersion
+FROM eventuateio/eventuate-examples-docker-images-java-example-base-image:$baseImageVersion
+COPY build/libs/customers-service.jar service.jar
+HEALTHCHECK --start-period=30s --interval=5s CMD curl -f http://localhost:8080/health || exit 1
```

Comment:
There is a micronaut version of the base image - BUILD-6

Reply:
I will update